### DISCUSSION TITLE
New to SnappyMail from Roundcube

### DISCUSSION THREAD

**Discussion by myfreexp** [the-djmaze/snappymail#608] [General]
Hi there,

I've been (and still am) a Roundcube user since 2012, using it as the one and only mail client for private and business purposes. As it looks to me that Roundcube is more and more coming closer to end-of-life, I've been looking for an alternative that is still and better supported and maintained. So I did a lot of reading and therefore came across SnappyMail, whose approach in terms of privacy (removal of Facebook, Google, etc.) I like and support.

Long ago, I was a developer and maintainer of a long outdated mail client and newsreader myself, but that was in the "good" ol' times of DOS/16 under Borland Pascal. So I do not have that much a clue of PHP and Javascript. The project was [FreeXP](https://de.wikipedia.org/wiki/CrossPoint#FreeXP_(DOS,_Windows)), a fork of "CrossPoint", which originally was a "Point program" for several mailbox networks such as FidoNet, Z-Netz, and MausNetz.

So much to my background. Currently, I'm running a test installation of SnappyMail, and coming from Roundcube, I do (of course) have a bunch of issues. My question is:

Shall I first list them here line by line and wait for comments before opening any issues on GitHub, or shall I do the latter right away, one for each issue?

Keep up the good work and best regards,
Michael

**Comment by MI-KY** [comment]
Hi Michael,

thanks for the link to FreeXP - very interesting 🙂.
Maybe you could list them here with some additional information  and then we could try to see what should really become a issue?
I'm also testing SnappyMail at the moment, so maybe I.can help you a little bit.
